### PR TITLE
feat: make the `brotli-size` dependency optional

### LINKED ISSUE #8: Usage for apps not using Brotli compression
Thanks for the great addon!

Could there be a way to make `brotli-size` an optional dependency for those that only use `gzip` exclusively in their projects? This dep pulls in `iltorb` which requires native OS bindings, and while this is possible to set up, it requires extra effort to configure for CI for something that isn't relevant for the consuming app.

Similar issues: https://github.com/siddharthkp/bundlesize/issues/202

### PR DESCRIPTION
**Pull request by jbailey4**
Closes #8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,6 @@
     "test": "mocha node-tests/**/*-test.js --recursive"
   },
   "dependencies": {
-    "brotli-size": "^0.0.3",
```
Comment (simonihmig) [inline package.json]: The brotli tests are failing now, see https://travis-ci.com/kaliber5/ember-cli-bundlesize/builds/111450101.

Please add that package to `devDependencies` for our local tests.

File: lib/helpers/compressed-size.js [hunk 2]
```diff
@@ -8,6 +8,17 @@ module.exports = function getCompressedSize(data, compression) {
       size = gzip.sync(data);
       break;
     case 'brotli':
+      console.log('BROTLI !!');
+      try {
+        brotli = require('brotli-size');
+        console.log(brotli);
+      } catch (error) {
+        console.log(error);
```
Comment (simonihmig) [inline lib/helpers/compressed-size.js]: Please remove all `console.log` output.

File: lib/helpers/compressed-size.js [hunk 3]
```diff
@@ -8,6 +8,17 @@ module.exports = function getCompressedSize(data, compression) {
...
+        throw new Error(
```
Comment (simonihmig) [inline lib/helpers/compressed-size.js]: Please add this pkg to `dependencies` and throw a `SilentError` instead, which will have some better UX (no stack trace): https://github.com/ember-cli/silent-error

File: README.md [hunk 4]
```diff
@@ -15,6 +15,13 @@ ember install ember-cli-bundlesize
 
 Make sure to `git add` the added `config/bundlesize.js` file!
 
+#### Optional dependency
+
+If you're app uses brotli compression and the `brotli` option is set in your configuration (i.e `compression: 'brotli'`), you must install the optional dependency: `brotli-size`.
```
Comment (simonihmig) [inline README.md]:
```suggestion
If your app uses Brotli compression and the `brotli` option is set in your configuration (i.e `compression: 'brotli'`), you must install the optional dependency: `brotli-size`.
```

**Comment by jbailey4** [comment]
@simonihmig Addressed the feedback and build is now passing. Ready for another review!

**Comment by simonihmig** [comment]
Published as 0.2.0! 🎉

**Pull request merged**